docs/tutorials/excel-tutorial.md

```diff
@@ -505,7 +505,7 @@ In this step of the tutorial, you'll add a button to the ribbon that toggles wor
 
 1. Open the manifest file **./manifest.xml**.
 
-1. Locate the `<Control>` element. This element defines the **Show Taskpane** button on the **Home** ribbon you have been using to launch the add-in. We're going to add a second button to the same group on the **Home** ribbon. In between the closing `</Control>` tag and the closing `</Group>` tag, add the following markup.
+1. Locate the **\<Control\>** element. This element defines the **Show Taskpane** button on the **Home** ribbon you have been using to launch the add-in. We're going to add a second button to the same group on the **Home** ribbon. In between the closing **</Control>** tag and the closing **</Group>** tag, add the following markup.
```

Comment:
I think these two closing tags need to be escaped, they are missing from the staging article: 
<img width="698" alt="image" src="https://user-images.githubusercontent.com/7977793/177427945-c72d3267-3402-4e79-8960-1a2b9495eb07.png">
Perhaps like this? 
```suggestion
1. Locate the **\<Control\>** element. This element defines the **Show Taskpane** button on the **Home** ribbon you have been using to launch the add-in. We're going to add a second button to the same group on the **Home** ribbon. In between the closing **\</Control\>** tag and the closing **\</Group\>** tag, add the following markup.
```